sklearn/inspection/_partial_dependence.py

```diff
@@ -253,10 +263,17 @@ def partial_dependence(
         also to generate values for the complement features when the
         `method` is 'brute'.
 
-    features : array-like of {int, str}
+    features : array-like of {int, str} or tuple of 2 {int, str}
         The feature (e.g. `[0]`) or pair of interacting features
         (e.g. `[(0, 1)]`) for which the partial dependency should be computed.
 
+    is_categorical : array-like of bool or tuple of 2 bool, default=None
+        Specifies if the `features` provided are categorical or continuous.
+        When a feature is declared as categorical, the grid values will be
```

Comment:
Since it's a boolean I would write
```suggestion
        Specifies whether the `features` provided are categorical or not.
        When a feature is declared as categorical, the grid values will be
```